delivery_roulier_laposte_fr/tests/test_laposte_label_dom.py

```diff
@@ -2,14 +2,19 @@
 
 from vcr_unittest import VCRMixin
 
+from odoo import tools
+
 from odoo.addons.base_delivery_carrier_label.tests import carrier_label_case
 
 
 class LaposteLabelDomCase(VCRMixin, carrier_label_case.CarrierLabelCase):
     def setUp(self, *args, **kwargs):
         # need it to be defined before super to avoid failure in _hide_sensitive_data
         self.account = False
-        super().setUp(*args, **kwargs)
+        # the cassettes for this case contain some big requests
+        # that are causing travis build to fail when logged
+        with tools.mute_logger("vcr.cassette"):
```

Comment:
you can decorate the whole class :wink:

Reply:
hehe but I wanted to limit the impact of this mute to the only problematic part: https://github.com/kevin1024/vcrpy/blob/master/vcr/cassette.py#L228